core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -787,4 +788,18 @@ public OptimizerConfig setUseCostBasedPartitioning(boolean useCostBasedPartition
         this.useCostBasedPartitioning = useCostBasedPartitioning;
         return this;
     }
+
+    @Min(0)
+    public int getPushFilterIntoValuesMaxRowCount()
+    {
+        return pushFilterIntoValuesMaxRowCount;
+    }
+
+    @Config("optimizer.push-filter-into-values-max-row-count")
```

Comment:
We're missing documentation for this one
fyi @mosabua

Reply:
https://github.com/trinodb/trino/pull/24373

but I need more info for the description and the description below should be fixed.. it makes no sense to me yet. Once you provide more info in the PR @kasiafi and @raunaqmorarka I can update the code annotation as well there